plugin-infra/go-plugin-access/src/com/thoughtworks/go/plugin/access/pluggabletask/PluggableTaskConfigStore.java

```diff
@@ -18,8 +18,8 @@
 
 import com.thoughtworks.go.plugin.access.config.PluginPreferenceStore;
 
-public final class PluggableTaskConfigStore extends PluginPreferenceStore<TaskPreference> {
```

Comment:
As @arvindsv mentioned, I needed to mock and hence had to remove final keyword.
Sorry for not replying earlier.